### ISSUE TITLE
Dashboard: Revisions tab - Two compare buttons appear on the page but only one is included in the instruction

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#115]
URL: https://hcdev.openplus.ca/en/node/60168/revisions
User Agent: Chrome
Language: English

Issue Description: There are two "Compare selected revisions" buttons on the page, but the instruction text only contains one that is located after the table "To compare two revisions, use the radio buttons from column A and B and press the compare button at the bottom of the table."
![image](https://user-images.githubusercontent.com/88776404/137136717-bed28e3a-98c2-4baa-8c2d-d759d7df6add.png)

Steps to reproduce:

1. Log into the Dashboard
2. Action on the "Build"
3. Select "Revisions" tab
4. Visual check

Expected results: The two "Compare selected revisions" buttons should be mentioned in the instruction text or only keep one button.

**Comment by echouin** [comment]
@XiuJuanXia From OP: ﻿this is related to the Diff module. There isn't much we can do about it unless you want us to rewrite the module.

**Comment by davidgotlieb** [comment]
Modified the instructions

**Comment by XiuJuanXia** [comment]
There is a minor issue in the modified instructions:

Use the radio buttons in the table below to select two revisions to compare. Then click the "Compare selected revisions" button to generate the comparison.

The Word "click" should be avoided. Perhaps use word "activate" for buttons.

**Comment by echouin** [comment]
@XiuJuanXia Can you give me the link to where you see this?

**Comment by XiuJuanXia** [comment]
@echouin, here is the link: https://rsams.dev.cloud.openplus.ca/en/node/64010/revisions

**Comment by echouin** [comment]
@XiuJuanXia 
OpenPlus has reported the issue to Drupal: ﻿https://www.drupal.org/project/diff/issues/3269171. Updated the entry.

**Comment by 40999221** [comment]
Comment from echouin last week (week of 22nd July, 2024):

"Not critical. Most likely a WxT issue. Might be resolved in a future update."

Please refer Github link from description for past communications.
